trendmicro_deepsecurity/plugin.spec.yaml

```diff
@@ -57,7 +57,7 @@ actions:
         type: "[]string"
         description: CVEs to protect against
         required: true
-        example: ["CVE-2005-0045","CVE-2014-0160","CVE-2017-0144","CVE-1337-1337"]
+        example: ['CVE-2005-0045','CVE-2014-0160','CVE-2017-0144','CVE-1337-1337']
```

Comment:
These should be double-quotes. If it generates it oddly in help.md  then update help.md after generation

Reply:
I have some issues with the generator. I'll fix it manually and report the error